game.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <link href="https://fonts.googleapis.com/css?family=Diplomata+SC|Griffy|Roboto" rel="stylesheet"/>
+    <link href="style.css" rel="stylesheet"/>
+    <title>
+    </title>
+  </head>
+  <body>
+
+<h1>Header</h1>
+<section>
+  <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+
+      <a href="index.html"><li>BACK TO INDEX</li></a>
+
+  </p>
+</section>
+    <script type= "text/javascript" src="app.js"></script>
```

Comment:
I like to put white space after my static markup and before my script tags, and then no whitespace between the scripts and the body.  I do this to indicate that the script tags aren't a part of the display markup.  it is really personal preference though.